global.json

```diff
@@ -0,0 +1,14 @@
+{
+  "sdk": {
+    "version": "7.0.100",
+    "allowPrerelease": true,
+    "rollForward": "latestPatch"
```

Comment:
We can adjust this to what makes the local development experience best if this is proven to be problematic.